packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -20,12 +13,12 @@ import { updateFirefoxExtensionLink } from '../../lib/constants';
 import UpvoteIcon from '../icons/Upvote';
 import CommentIcon from '../icons/Discuss';
 import InteractionCounter from '../InteractionCounter';
-import SettingsContext from '../../contexts/SettingsContext';
-import { AlertColor, AlertDot } from '../AlertDot';
 import { checkIsExtension } from '../../lib/func';
+import { UserPostVote } from '../../graphql/posts';
+import InteractivePopup, { InteractivePopupPosition } from './InteractivePopup';
+import { Origin } from '../../lib/analytics';
 
-interface ChangelogTooltipProps<TRef> extends BaseTooltipProps {
-  elementRef: MutableRefObject<TRef>;
+interface ChangelogTooltipProps extends BaseTooltipProps {
```

Comment:
I don't think the BaseTooltipProps are extended anymore right?

Reply:
True, will remove, thank u!